pages/docs/reference/whatsnew14.md

```diff
@@ -6,6 +6,19 @@ title: "What's New in Kotlin 1.4"
 
 # What's New in Kotlin 1.4
 
+## Delegated properties improvements
+
+In 1.4, we add new features to improve your experience with delegated properties in Kotlin:
+- now a property can be delegated to another property
```

Comment:
- Now a property can be delegated to another property.
- A new interface `PropertyDelegateProvider` helps create delegate providers in a single declaration.
- `ReadWriteProperty` now extends `ReadOnlyProperty` so you can use both of them for read-only properties.